File: 1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence/1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence.py
class Solution(object):
    def isPrefixOfWord(self, sentence, searchWord):
        """
        :type sentence: str
        :type searchWord: str
        :rtype: int
        """
        lister = sentence.split(" ")
        print(lister)
        
        for word in range(len(lister)):
            check = True
            if len(searchWord) <= len(lister[word]):
                for i in range(len(searchWord)):
                    if lister[word][i] != searchWord[i]:
                        check = False
            else:
                check = False
            if check == True:
                return word + 1
        return -1

File: 1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence/1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence.java
class Solution {
    public int isPrefixOfWord(String sentence, String searchWord) {
        String[] splitter = sentence.split(" ");
        for(int i = 0; i < splitter.length; i++){
            if(splitter[i].length() >= searchWord.length()){
            boolean match = true;
            for(int x = 0; x < searchWord.length(); x++){
                if(splitter[i].charAt(x) != searchWord.charAt(x)){
                    match = false;
                }
            };
            
            if(match){
                return i + 1;
            }
            }
        }
        
        return -1;
        
    }
}